sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -45,11 +45,13 @@ first_seen AS (
 activations AS (
   SELECT
     client_id,
-    ARRAY_AGG(activated ORDER BY submission_date DESC)[SAFE_OFFSET(0)] > 0 AS activated
+    ARRAY_AGG(activation.activated ORDER BY activation.submission_date DESC)[SAFE_OFFSET(0)] > 0 AS activated,
   FROM
-    fenix.new_profile_activation
+    fenix.new_profile_activation AS activation
+  INNER JOIN `moz-fx-data-shared-prod.fenix_derived.firefox_android_clients_v1` --excluding clients in activations but not in baseline ping due to shredder
```

Comment:
Probably easier later to find the comment at the top of the subquery?

Reply:
Should `activation` be going through shredder as well?

Reply:
It seems so because it has client_id. I'd suggest doing that in a separate PR and then we could remove the inner join. To my knowledge, adding it to [this script ](https://github.com/mozilla/bigquery-etl/blob/2a243c483d221d6a28ca5da7cf6975e9b159411e/bigquery_etl/shredder/config.py#L153) should be enough. What do you think Winnie?